kafka-ui-react-app/src/components/common/Pagination/__tests__/PageControl.spec.tsx

```diff
@@ -1,32 +1,45 @@
 import React from 'react';
-import { mount, shallow } from 'enzyme';
+import { shallow } from 'enzyme';
 import { StaticRouter } from 'react-router';
 import PageControl, {
   PageControlProps,
 } from 'components/common/Pagination/PageControl';
+import { ThemeProvider } from 'styled-components';
+import theme, { Colors } from 'theme/theme';
+import { screen } from '@testing-library/react';
+import { render } from 'lib/testHelpers';
 
 const page = 138;
 
 describe('PageControl', () => {
-  const setupWrapper = (props: Partial<PageControlProps> = {}) => (
-    <StaticRouter>
-      <PageControl url="/test" page={page} current {...props} />
-    </StaticRouter>
-  );
+  const setupComponent = (props: Partial<PageControlProps> = {}) =>
+    render(
+      <ThemeProvider theme={theme}>
```

Comment:
Pls check `render` in `lib/testHelpers` method. there is no need to wrap it to ThemeProvider again